zemljevid.js

```diff
@@ -103,6 +112,28 @@ animiraj = function(enMarker, newx, newy) {
     }, 0);
 }
 
+update_smer = function(enMarker, newx, newy, newbear) {
+    var p = 0; // potek interpolacije
+
+    var x = enMarker.getLatLng().lat;
+    var y = enMarker.getLatLng().lng;
+    var pr = setInterval(function() {
+        if (p < 1) {
+            enMarker.setLatLng(new L.LatLng(cosp(x, newx, p), cosp(y, newy, p)));
+            p += 0.1;
+            var oldTransform = enMarker._icon.style.transform;
+            enMarker._icon.style.transform = `${oldTransform} rotate(${newbear}deg)`;
```

Comment:
Leaflet refreshes the transform values on zoom. Hence, this value is overwritten and all markers start pointing upwards.

I will fix this by using custom-modified Leaflet code. For now, please use the following code instead (I'll add the modified Leaflet code later):
`enMarker["_icon"].dataset.orientacija = newbear;`